swift-sdk/IterableAPI.swift

```diff
@@ -46,45 +46,7 @@ import UIKit
         _ = internalImplementation?.start()
     }
     
-    /**
-     The email of the logged in user that this IterableAPI is using
-     */
-    public static var email: String? {
-        get {
-            return internalImplementation?.email
-        } set {
-            internalImplementation?.email = newValue
-        }
-    }
-    
-    /**
-     The userId of the logged in user that this IterableAPI is using
-     */
-    public static var userId: String? {
-        get {
-            return internalImplementation?.userId
-        } set {
-            internalImplementation?.userId = newValue
-        }
-    }
-    
-    /**
-     The userInfo dictionary which came with last push.
-     */
-    public static var lastPushPayload: [AnyHashable: Any]? {
-        return internalImplementation?.lastPushPayload
-    }
-    
-    /**
-     Attribution info (campaignId, messageId etc.) for last push open or app link click from an email.
-     */
-    public static var attributionInfo: IterableAttributionInfo? {
-        get {
-            return internalImplementation?.attributionInfo
-        } set {
-            internalImplementation?.attributionInfo = newValue
-        }
-    }
+    // MARK: - API Request Calls
     
```

Comment:
These properties should be at top. So the class definition should usually be like

Class {
    public property
    public property
    public methods
}